src/main/java/uk/gov/ida/verifyserviceprovider/factories/VerifyServiceProviderFactory.java

```diff
@@ -13,8 +13,8 @@
 import uk.gov.ida.verifyserviceprovider.factories.saml.ResponseFactory;
 import uk.gov.ida.verifyserviceprovider.healthcheck.MetadataHealthCheck;
```

Comment:
`import uk.gov.ida.verifyserviceprovider.healthcheck.MetadataHealthCheck;` appears to be unused
along with these private methods
getHubMetadataResolver()
getMsaMetadataResolver()